apps/spruce/codegen.ts

```diff
@@ -7,9 +7,11 @@ export const getConfig = ({
 }: {
   generatedFileName: string;
 } & Pick<CodegenConfig, "silent">): CodegenConfig => ({
-  documents: ["./src/**/*.ts", "./src/**/*.graphql", "./src/**/*.gql"].map(
-    (d) => resolve(process.cwd(), d),
-  ),
+  documents: [
+    "./src/**/!(*.test).ts",
```

Comment:
is it possible to remove this line? do we ever define GraphQL queries in TS?
```suggestion
```

Reply:
A query is defined in this [test](https://github.com/evergreen-ci/ui/pull/711/files#diff-6df8c6c4517927190e274ace0d2476e3c23872e58a757220f186643b4d9e2914R8)